### ISSUE TITLE
Lite mode additional info

### ISSUE THREAD

**Issue by pctechkid** [maple3142/GDIndex#55]
Someone please add this feature for LITE mode as I prefer lite mode than the original theme. It is very useful for file/folder details.
![image](https://user-images.githubusercontent.com/27243557/75631362-31fa6a80-5c2d-11ea-826b-5a0b9a895550.png)

**Comment by maple3142** [comment]
Lite mode is specifically designed to be similar to nginx's default directory listing. Otherwise, I am not sure that can be recursively downloaded by wget.

**Comment by CatTheHacker** [comment]
Seems to be working fine for me. [example](https://msdn.hackerc.at)

**Comment by pctechkid** [comment]
> Seems to be working fine for me. [example](https://msdn.hackerc.at)

Yeah, I want my lite mode to have info such as date modified and file size just like yours. Can you please help me?

**Comment by pctechkid** [comment]
> Seems to be working fine for me. [example](https://msdn.hackerc.at)

@CatTheHacker 
Yes, I want my lite mode to have info such as date modified and file size just like yours. Can you please help me?

**Comment by CatTheHacker** [comment]
I only modified my `worker.js` to do that stuff as I'm not really good at JS but basically you want to modify line [648](https://github.com/maple3142/GDIndex/blob/84218d1cdead8e09599c7d22d73c23df682b2727/worker/dist/worker.js#L648) only.
Original:
```javascript
        fileht += `<li><a href="${p + (isf ? '/' : '')}">${f.name}</a></li>`;
```
to something like this:
```javascript
        fileht += `<li><a href="${p + (isf ? '/' : '')}">${f.name}</a> &nbsp; ${f.modifiedTime} &nbsp; ${f.size}</li>`;
```

for date and size I have small helper functions converting them:
```javascript
function formatBytes(bytes, decimals = 2) {
    if (bytes === 0) return '0 Bytes';

    const k = 1024;
    const dm = decimals < 0 ? 0 : decimals;
    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];

    const i = Math.floor(Math.log(bytes) / Math.log(k));

    return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
  }

  function formatDate(date) {
    date = date.split('T');
    date[1] = ((date[1].split('Z')[0]).split('.'))[0];
    return '' + date[0] + ' ' + date[1];
  }
```
with that it would be something like:
```javascript
        fileht += `<li><a href="${p + (isf ? '/' : '')}">${f.name}</a> &nbsp; ${formatDate(f.modifiedTime)} &nbsp; ${formatBytes(f.size)}</li>`;
```
🐱‍💻

**Comment by pctechkid** [comment]
> I only modified my `worker.js` to do that stuff as I'm not really good at JS but basically you want to modify line [648](https://github.com/maple3142/GDIndex/blob/84218d1cdead8e09599c7d22d73c23df682b2727/worker/dist/worker.js#L648) only.
> Original:
> 
> ```js
>         fileht += `<li><a href="${p + (isf ? '/' : '')}">${f.name}</a></li>`;
> ```
> 
> to something like this:
> 
> ```js
>         fileht += `<li><a href="${p + (isf ? '/' : '')}">${f.name}</a> &nbsp; ${f.modifiedTime} &nbsp; ${f.size}</li>`;
> ```
> 
> for date and size I have small helper functions converting them:
> 
> ```js
> function formatBytes(bytes, decimals = 2) {
>     if (bytes === 0) return '0 Bytes';
> 
>     const k = 1024;
>     const dm = decimals < 0 ? 0 : decimals;
>     const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
> 
>     const i = Math.floor(Math.log(bytes) / Math.log(k));
> 
>     return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
>   }
> 
>   function formatDate(date) {
>     date = date.split('T');
>     date[1] = ((date[1].split('Z')[0]).split('.'))[0];
>     return '' + date[0] + ' ' + date[1];
>   }
> ```
> 
> with that it would be something like:
> 
> ```js
>         fileht += `<li><a href="${p + (isf ? '/' : '')}">${f.name}</a> &nbsp; ${formatDate(f.modifiedTime)} &nbsp; ${formatBytes(f.size)}</li>`;
> ```
> 
> 🐱‍💻

@CatTheHacker Thanks a lot. It worked but one more issue I encountered, the files itself are displaying fine, but the folders are not. They display NaN undefined, how to fix this?

![image](https://user-images.githubusercontent.com/27243557/79476635-ced96180-803b-11ea-9799-048ae5cabe9a.png)

**Comment by CatTheHacker** [comment]
`${(isf ? '-' : formatBytes(f.size))}`

**Comment by pctechkid** [comment]
> ```js
> ${(isf ? '-' : formatBytes(f.size))}
> ```

Thank youuuu. It's now working and looked as I wanted it to be. I owe you one.

**Comment by pctechkid** [comment]
@CatTheHacker oh, one thing I didn't notice. The modified time shown is a bit off, it's behind at least 8hrs. I tried modifying it but it didn't turn out right :(

**Comment by CatTheHacker** [comment]
Most likely because the time is in UTC format and my date helper does not convert the date to the local timezone.

**Comment by pctechkid** [comment]
> Most likely because the time is in UTC format and my date helper does not convert the date to the local timezone.

Is it not possible to just literally add 8hrs in the function or something like that? it doesnt have to be a timezone conversion as it sounds more complicated.

**Comment by CatTheHacker** [comment]
Something like this I guess? idk if it works
```javascript
function formatDate(date) {
    date = date.split('T');
    date[1] = ((date[1].split('Z')[0]).split('.'))[0];
    date[1] = date[1].split(':');
    date[1][0] = date[1][0] + 8;
    date[1] = date[1].join(':');
	return '' + date[0] + ' ' + date[1];
}
```

**Comment by pctechkid** [comment]
> Something like this I guess? idk if it works
> 
> ```js
> function formatDate(date) {
>     date = date.split('T');
>     date[1] = ((date[1].split('Z')[0]).split('.'))[0];
>     date[1] = date[1].split(':');
>     date[1][0] = date[1][0] + 8;
>     date[1] = date[1].join(':');
> 	return '' + date[0] + ' ' + date[1];
> }
> ```

unfortunately, it does not add 8hrs , it only adds '8' character on the hours place making it 008:00

**Comment by CatTheHacker** [comment]
JavaScript do be like that ¯\_(ツ)_/¯

**Comment by pctechkid** [comment]
> JavaScript do be like that ¯\_(ツ)_/¯

Well yeah. One last question sir, how do I change the list style type of <li> whenever it is a folder, folder should be the icon, and when it's a file, a file icon. Just like your [example](https://msdn.hackerc.at/)

**Comment by cronopsxploit** [comment]
@maple3142 
I have these 2 recurring errors then I need to keep updating the page to look like the content again
how am i new can someone tell me how to solve?
![1](https://user-images.githubusercontent.com/68617575/106170285-b1a6f700-6166-11eb-8887-dd9cee374aaf.png)
![2](https://user-images.githubusercontent.com/68617575/106170289-b23f8d80-6166-11eb-9715-9f46863c5e61.png)

I followed all the steps, described in git, I used your automatic token generator.
it works however if I enter and leave a folder sometimes it needs to be reloading the page until the folders appear again.
I went to your test drive and try to reproduce the error by updating the page frequently, but in the case of yours it didn't work out as I can be more specific so that I help resolve
